### PR TITLE
Try catch property

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- fixes this https://sentry.io/organizations/posthog/issues/2135731045/?project=1899813
- there's a dashboard with a malformed property dict that's causing this issue

*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: posthog/api/dashboard.py [hunk 1]
```diff
@@ -208,8 +208,6 @@ def get_result(self, dashboard_item: DashboardItem):
     def get_last_refresh(self, dashboard_item: DashboardItem):
         if self.get_result(dashboard_item):
             return dashboard_item.last_refresh
-        dashboard_item.last_refresh = None
-        dashboard_item.save()
```
Comment (EDsCODE) [inline posthog/api/dashboard.py]: not convinced this is the best place for this but I left it in for now

**Review by macobo — APPROVED** [review]
I wonder if we should create a test under e.g. dashboards for this.

**Comment by EDsCODE** [comment]
yeah hold up somehow blew something up too

**Comment by macobo** [comment]
LGTM

**Pull request merged**